api/src/main/java/org/pmiops/workbench/elasticsearch/ElasticSearchService.java

```diff
@@ -30,29 +30,26 @@ public ElasticSearchService(Provider<WorkbenchConfig> configProvider) {
 
   /**
    * This method will generate counts per the org.pmiops.workbench.model.SearchRequest using elasticsearch.
-   *
-   * @param userRequest
    */
-  public Long elasticCount(SearchRequest userRequest) {
+  public Long elasticCount() {
     //For now if enableElasticsearchBackend is true
     //we get the cluster name, status and log it
     try {
       ClusterHealthRequest healthRequest = new ClusterHealthRequest();
       ClusterHealthResponse response = client().cluster().health(healthRequest, RequestOptions.DEFAULT);
 
       log.info(String.format("Cluster Name: %s - Cluster Status: %s", response.getClusterName(), response.getStatus().name()));
+      log.info(String.format("Cluster Name: %s - Cluster Status: %s", response.getClusterName(), response.getStatus().name()));
```

Comment:
nit: duplicated logger line?

Reply:
Doh.. not sure what happened there. Done.